gamemodes/zombiesurvival/gamemode/init.lua

```diff
@@ -3256,6 +3256,13 @@ function GM:WeaponDeployed(pl, wep)
 	end
 end
 
+local function playerIsStuckInsideObj(plvel)
```

Comment:
what the literal fuck is this function?

Reply:
For some reason I thought that lua wouldn't understand comparing float numbers with different decimal cases even though technically they have the same value, dumb me lol